firebase-config/src/test/java/com/google/firebase/remoteconfig/internal/ConfigFetchHandlerTest.java

```diff
@@ -959,10 +981,26 @@ private void verifyETags(@Nullable String requestETag, String responseETag) thro
             /* lastFetchETag= */ eq(requestETag),
             /* customHeaders= */ any(),
             /* firstOpenTime= */ any(),
-            /* currentTime= */ any());
+            /* currentTime= */ any(),
+            /* customSignals= */ any());
     assertThat(sharedPrefsClient.getLastFetchETag()).isEqualTo(responseETag);
   }
 
+  private void verifyCustomSignals(Map<String, String> customSignals) throws Exception {
+    verify(mockBackendFetchApiClient)
```

Comment:
I would suggest that we assert that the custom signals in the fetch arguments is the same as what is set in the shared preferences (instead of passing it as an argument to this verify method). Optionally (even better IMO), inline this assertion in the test method itself.

Reply:
Updated it to assert that the custom signals in the fetch arguments is the same as what is set in the shared preferences.